packages/cli/src/commands/data-migrate.js

```diff
@@ -1,5 +1,5 @@
-export const command = 'dataMigrate <command>'
-export const aliases = ['dm']
+export const command = 'data-migrate <command>'
+export const aliases = ['dm', 'dataMigrate', 'datamigrate']
```

Comment:
see comment about `type-check`

Note: in the `yarn rw deploy <provider> [option]` command, we use `data-migrate` as the option for `dataMigrate`. Lulz...

Reply:
Exactly. That's what I'm confused by - yargs _must_ do something right?